src/translib/transformer/sw_portchannel.go

```diff
@@ -167,7 +173,7 @@ func can_configure_fallback(inParams XfmrParams) error {
            var mode string
            e = get_lag_type(inParams.d, &ifKey, &mode)
            if e == nil {
-               errStr := "Fallback option cannot be re-configured for an already existing PortChannel: " + ifKey
+               errStr := "Fallback option cannot be configured for an already existing PortChannel: " + ifKey
```

Comment:
Setting the same mode should be a no-op. So, why have we changed "re-configured" to "configured"

Reply:
This also gets called for portchannel that is already configured, not necessarily for updating the same mode, no-op case may need additional check.